lld/ELF/InputFiles.h

```diff
@@ -224,8 +228,10 @@ class ELFFileBase : public InputFile {
   StringRef stringTable;
   const void *elfShdrs = nullptr;
   const void *elfSyms = nullptr;
+  const void *elfPhdrs = nullptr;
```

Comment:
ELFFileBase is used by both .o and .so

move phdrs (image file only) to SharedFile and initialize them in SharedFile::parse